.evergreen/config.yml

```diff
@@ -860,19 +878,38 @@ axes:
           DEPENDENCIES: "lowest"
 
 buildvariants:
-# Test all PHP versions with latest-stable MongoDB and PHPC on Debian
+# Test all PHP versions with latest-stable MongoDB and PHPC, RHEL, and Ubuntu
 - matrix_name: "test-php-versions"
-  matrix_spec: { "os": "debian11", "mongodb-edge-versions": "latest-stable", "php-versions": "*", "driver-versions": "latest-stable" }
+  matrix_spec:
+    os:
+      - debian11
+      - debian10
+      - rhel90
+      # TODO: RHEL 8.x is not working yet
+      # - rhel82-arm64
+      # - rhel80
+      # TODO: RHEL 7.6 fails as it's not able to connect to pecl.php.net to install the library
+      # - rhel76
+      - ubuntu2204
+      - ubuntu2204-arm64
+      # TODO: KMS servers fail to start on Ubuntu 20.04
```

Comment:
Noted that we previously weren't testing on _any_ Ubuntu platforms. `ubuntu1804-arm64` was defined in the 'os` matrix but never used.

Do you plan on opening a "Build Failure" ticket for this? If so, let's add a ticket reference here.

I don't feel strongly about it as the OS matrix is more relevant for PHPC than PHPLIB.

Reply:
I'm happy to slim down the OS matrix as the platform is more relevant to PHPC than PHPLIB. For PHPLIB, I'd say the important systems are Ubuntu and Debian, as those are were most people run their applications.

An important difference are docker users, where Alpine is a popular choice, but that's not available. I'll note that the standard images (e.g. `php-cli`) are built on Debian, so we have those covered.

Reply:
FYI, [PHPC-1555](https://jira.mongodb.org/browse/PHPC-1555) is an old tracking ticket to add Alpine, which we had to resolve as "Won't Fix" due to missing support in Evergreen. If we can get Alpine working in GitHub actions then perhaps we can revisit this down the line.

Reply:
As per suggestions, I have slimmed down the OS matrix, omitting RHEL 7 and RHEL 8 completely as the platform doesn't have an impact on the library. We're testing on RHEL 9, which also doesn't have the certificate problem I saw on earlier RHEL versions.

Reply:
@alcaeus Will you be creating a ticket for "KMS servers fail to start on Ubuntu 20.04"?

Reply:
> @alcaeus Will you be creating a ticket for "KMS servers fail to start on Ubuntu 20.04"?

I wanted to grab a fresh copy of the error message, but the patch I created for that purpose succeeded: https://spruce.mongodb.com/version/64ad54d157e85a3f891c1afe/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

I'm now running a new patch on all Ubuntu 20.04 platforms - if that succeeds I'll add them in here; otherwise I'll create a ticket to have it looked at.